dist/i18n-react.js

```diff
@@ -199,17 +199,15 @@ var MDText = /** @class */ (function () {
             trans = (options && options.notFound !== undefined) ? options.notFound :
                 this.notFound !== undefined ? this.notFound :
                     key;
-            if (isFunction(trans)) {
-                trans = trans(key, options);
-            }
-            if (!isString(trans)) {
-                return trans;
-            }
         }
         if (isFunction(trans)) {
-            trans = trans(key, context);
+            var args = context === undefined ? options : context;
+            trans = trans(key, args);
+        }
+        if (isString(trans)) {
+            trans = this.format(trans, options);
         }
-        return this.format(trans, options);
+        return trans;
```

Comment:
Can you please squash commits next time for the pull request - I have trouble seeing what is new

Reply:
Sure, sorry for that.